lib/sycamore/sycamore/transforms/llm_query.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         llm_kwargs: dict = {},
         per_element: bool = True,
         element_type: Optional[str] = None,
+        table_cont: Optional[bool] = False,
```

Comment:
I think this makes more sense as a subclass? hmm, not sure. worth thinking through the factoring at least

Reply:
@dtecuci ?

Reply:
Decided to not have subclass